src/lazy/encoder/binary/v1_1/flex_uint.rs

```diff
@@ -36,7 +36,7 @@ pub struct FlexUInt {
 }
 
 impl FlexUInt {
-    fn new(size_in_bytes: usize, value: u64) -> Self {
+    pub(crate) fn new(size_in_bytes: usize, value: u64) -> Self {
```

Comment:
Thoughts on this? In a future PR I needed to do similar to FixedInt, and opted to just implement `From<i64>`. In retrospect implementing `From<u64>` here would probably be better, imo.

Reply:
Do you need to create an instance of `FlexUInt` from the reader other than what you can create using `FlexUInt::read()`?

Reply:
Yes. For simplicity, I handle the lengths of values via FlexUInt, so I need to create instances for when the header provides enough information for the length.

I probably should have provided more context. The question is more to gain consensus around whether `new` is sufficiently user-friendly for the crate. The `size_in_bytes` argument makes it seem a little more 'internal', which would make me think a `From<u64>` or similar added method would be a better alternative.

Reply:
I think it's fine as long as it's `pub(crate)`. There's not a reason for (most) users to manually create a type representing an encoding artifact. If we want to eventually make `Flex[U]Int` available to the public (say, for use in the `ion-cli`), we would add an experimental feature gate for it anyway.

Having `From<i64/u64>` is great for simplicity as long as the type stays out of the public API.